content/en/real_user_monitoring/dashboards/testing_coverage.md

```diff
@@ -0,0 +1,53 @@
+---
+title: RUM Application Testing Coverage Dashboard
+kind: documentation
+description: Learn about the out-of-the-box RUM application testing coverage dashboard.
+further_reading:
+- link: '/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM & Session Replay'
+- link: '/synthetics/browser_tests'
+  tag: 'Documentation'
+  text: 'Learn about Synthetic browser tests'
+---
+
+## Overview
+
+The [application testing coverage dashboard][1] uses data collected from [RUM & Session Replay][2] and results from Synthetic [browser tests][3] to provide insights about the overall testing coverage for your application. For more information, see [Explore RUM & Session Replay in Synthetics][4].
```

Comment:
Data comes from RUM only. no need for Session replay. I think it is important to make the difference as Session replay is made as a premium https://www.datadoghq.com/pricing/?_gl=1*cx28xz*_gcl_aw*R0NMLjE2NTMzOTMyMTUuQ2owS0NRandoTEtVQmhEaUFSSXNBTWFUTG5FOUlZaWRURVgxRmc5SGNnNmpCVkYxR0I1YVRrN1ZGU3pWcXphN0FPZDNUMHh2bjdyRzBnZ2FBdlljRUFMd193Y0I.*_ga*MTIyOTE5NDU0LjE2MjM2ODExMzI.*_ga_KN80RDFSQK*MTY1NjUxNDExOC45MTIuMS4xNjU2NTE0MTg4LjYw&product=real-user-monitoring--session-replay#real-user-monitoring--session-replay